shared/middleware/helpers/redis_table.rb

```diff
@@ -18,24 +17,21 @@ class NotFound < Sinatra::NotFound
   # Constructs the redis table.
   #
   # @param [Redis] redis The Redis client.
-  # @param [PubSubApi] pub_sub_api An optional PubSub API implementation
   # @param [String] shard_id
   # @param [String] table_name
-  def initialize(redis, pub_sub_api, shard_id, table_name)
-    @pub_sub_api = pub_sub_api
-
+  def initialize(redis, shard_id, table_name)
     @shard_id = shard_id
     @table_name = table_name
 
     # A counter key in the underlying RedisPropertyBag, used to generate
-    # asecending row ids.
+    # ascending row ids.
     @row_id_key = "#{table_name}_row_id"
 
     # A redis property bag for storing all tables in the shard.
     @props = RedisPropertyBag.new(redis, shard_id)
   end
 
-  # Inserts a new row, notifying other clients using the pubsub service.
+  # Inserts a new row.
```

Comment:
Regarding the publish on cascading delete: I believe that at this time we do not want to do that.  The reason is that we are currently publishing invalidations, not deltas, and we have already sent an invalidation for the table where the cascading delete as happen.  At best the client will ignore or coalesce the repeated invalidation for the same table, and at worst it will do an unnecessary refresh, so it seems there is no real benefit to sending that invalidation.